utils/jera_database_provider/__brick__/database/setup/database_provider.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:path/path.dart';
+import 'package:sqflite/sqflite.dart';
+
+import '../table/table.dart';
+
+class DatabaseProvider {
+  DatabaseProvider._();
+
+  static DatabaseProvider instance = DatabaseProvider._();
+
+  late Database _db;
+
+  bool _isInitialized = false;
+
+  //TODO adicione suas tabelas aqui
+  List<Table> tables = [];
```

Comment:
Essa lista pode ser `final`:

```suggestion
  final tables = <Table>[];
```